### PR TITLE
Nocross nomusl

### PR DESCRIPTION
**Pull request by iav**
Docker chain copied from "dev" but removed cross-compilation to musl and final lemmy container based on debian.
This chain tested and run on x64 (in vitualbox), Raspberry Pi (2B,3B+), Rock64 (I have to lower maximal CPU rate to 1GHz due stability issues on rust compilation, without this random compile errors occurs in random places).

### CODE REVIEW

File: docker/nocross/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  && echo 'fn main() { println!("Dummy") }' > ./src/bin/main.rs 
+
+RUN cargo build
+RUN rm -f ./target/debug/deps/lemmy_server*
+COPY server/src ./src/
+COPY server/migrations ./migrations/
+
+# build for release
+#RUN cargo build --frozen --release
+RUN cargo build
```
Comment (dessalines) [inline docker/nocross/Dockerfile]: Does frozen release not work here?

File: docker/nocross/Dockerfile.org [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# Copy resources
+COPY --from=rust /app/server/target/x86_64-unknown-linux-musl/release/lemmy_server /app/lemmy
+COPY --from=node /app/ui/dist /app/dist
+RUN addgroup -g 1000 lemmy
+RUN adduser -D -s /bin/sh -u 1000 -G lemmy lemmy
+RUN chown lemmy:lemmy /app/lemmy
+USER lemmy
+EXPOSE 8536
+CMD ["/app/lemmy"]
```
Comment (dessalines) [inline docker/nocross/Dockerfile.org]: Whats the point of this file? It looks identical to the one in docker/dev, has all the musl stuff here.

File: docker/nocross/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+git push
+
+# Rebuilding docker
+./docker_update.sh
+docker tag dev_lemmy:latest dessalines/lemmy:$new_tag
+docker push dessalines/lemmy:$new_tag
+
+# Pushing to any ansible deploys
+cd ../../ansible
+ansible-playbook lemmy.yml
```
Comment (dessalines) [inline docker/nocross/deploy.sh]: This file duped isn't necessary, you can remove it here, and I can make sure if this works to build and push the other one to docker-hub with the deploy script in the other folder. Maybe with a docker tage of something like `v0.1.0-nocross` or something.

File: docker/nocross/Dockerfile (hunk 1 above)
Comment (iav) [reply docker/nocross/Dockerfile]: I have to try.

File: docker/nocross/Dockerfile.org (hunk 2 above)
Comment (iav) [reply docker/nocross/Dockerfile.org]: Yes. I  remove it.

File: docker/nocross/deploy.sh (hunk 3 above)
Comment (iav) [reply docker/nocross/deploy.sh]: I remove it.

File: docker/nocross/Dockerfile [hunk 4]
```diff
@@ -28,7 +28,7 @@ COPY server/migrations ./migrations/
 
 # build for release
 #RUN cargo build --frozen --release
-RUN cargo build
+RUN cargo build --frozen
```
Comment (dessalines) [inline docker/nocross/Dockerfile]: Why does release not work?

File: docker/nocross/Dockerfile (hunk 1 above)
Comment (iav) [reply docker/nocross/Dockerfile]: I try, --frozen works. I put it back.

File: docker/nocross/Dockerfile (hunk 4 above)
Comment (iav) [reply docker/nocross/Dockerfile]: "--release" takes about 2 times more time, and want more RAM. I have to add swap space over defaul for raspbian  on SBC with 1 GB RAM, but without "--release" this swap required for short time. With "--release" build runs more then 90 minutes on RPi 2B. That's my reason.
"--release" will work, but takes very much time on SBC.

Comment (dessalines) [reply docker/nocross/Dockerfile]: I'll try this build on my machine and make sure it caches and such.

**Pull request merged**